tests/services/test_links.py

```diff
@@ -10,52 +10,51 @@
 """Test links."""
 
 import pytest
-from flask_principal import Identity
+from marshmallow import Schema, fields
 
 
-@pytest.fixture()
-def identity_no_need():
-    """Simple identity fixture without needs."""
-    i = Identity(1)
-    return i
```

Comment:
We should keep this around to test permission and links even if that tests is skipped for now.